README.md

```diff
@@ -13,7 +13,7 @@
 
 **Documentation**: [https://www.uvicorn.org](https://www.uvicorn.org)
 
-**Requirements**: Python 3.6+ (For Python 3.5 support, install version 0.8.6.)
+**Requirements**: Python 3.7+ (For Python 3.6 support, install version 0.15.0.)
```

Comment:
I've removed the 3.5 info. ~15 daily downloads for Python 3.5 and this information is only relevant if you want to start using uvicorn on a Python 3.5 project.

Reference: https://pypistats.org/packages/uvicorn